### PR TITLE
Adding more info about shifting location in China

### PR DESCRIPTION
**Pull request by langsmith**
This pr adds more adjustments to the code and writing regarding the `LocationComponent` and shifting device location in China.

cc @Chaoba @LukasPaczos @chriswu42 @springmeyer 

screen shot 2018-11-06 at 11 32 26 am

### CODE REVIEW

File: src/pages/plugins/overview/china.md [hunk 1]
```diff
@@ -123,43 +123,35 @@ try {
 
 ## Shifting location
 
-[Read how to use the Mapbox `LocationEngine` in the Mapbox Core Libraries for Android](https://www.mapbox.com/android-docs/core/overview/#locationengine)
+Rather than dealing with raw coordinate values, the `ShiftLocation` class and its `shift` method handle `Location` objects. Showing a device's current location via [the Maps SDK's `LocationComponent`](https://www.mapbox.com/android-docs/maps/overview/location-component/) is one of the most common use cases for using the `ShiftLocation` class.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/china.md]: dealing --> working ?

Comment (colleenmcginnis) [reply src/pages/plugins/overview/china.md]: `https://www.mapbox.com/android-docs/maps/overview/location-component/` --> `/android-docs/maps/overview/location-component/` (that way you don't get pushed out to the production version when you're developing locally.)

File: src/pages/plugins/overview/china.md [hunk 2]
```diff
@@ -123,43 +123,35 @@ try {
...
 
-When a new location update occurs, you'll need to manually feed the `location` object into the plugin's `ShiftLocation` class' `shift()` method. Rather than dealing with raw coordinate values, this method and class handles `Location` objects. For now, the best way to do this is to create your own `LocationEngine` which extends another and listens for location updates. When the update occurs, send the `Location` object through the shift module and have the `locationEngine` provide the modified location.
+[After setting up your own Mapbox `LocationEngine`](https://www.mapbox.com/android-docs/core/overview/#locationengine), you'll eventually override the `onLocationChanged()` method. When a new location update occurs, you'll need to manually feed the unshifted `Location` object into the `ShiftLocation` class' `shift()` method. `shift()` returns a `Location` object which you can now use however you'd like. If you pass the shifted `Location` object to `forceLocationUpdate()`, the `LocationComponent` will place the device location "puck" in the correct location.
```
Comment (colleenmcginnis) [inline src/pages/plugins/overview/china.md]: `which` --> `, which` ?

Comment (colleenmcginnis) [inline src/pages/plugins/overview/china.md]: You can also use a relative link here (`/android-docs/core/overview/#locationengine`)

File: src/pages/plugins/overview/china.md (hunk 1 above)
Comment (langsmith) [reply src/pages/plugins/overview/china.md]: Fixed

File: src/pages/plugins/overview/china.md (hunk 2 above)
Comment (langsmith) [reply src/pages/plugins/overview/china.md]: Fixed

Comment (langsmith) [reply src/pages/plugins/overview/china.md]: fixed

**Pull request merged**